tests/integration/conftest.py

```
@@ -0,0 +1,432 @@
+    kafka_container: str,
+    postgres_container: str,
+    localstack_container: dict,
+    mock_jwt_server: str,
+) -> Generator[Callable[[Dict[str, Any]], Dict[str, Any]], None, None]:
+    """Create lambda_handler with real container backends."""
+    # Set environment variables for the Lambda.
+    os.environ["LOG_LEVEL"] = "DEBUG"
+    os.environ["AWS_ENDPOINT_URL"] = localstack_container["url"]
+    os.environ["AWS_DEFAULT_REGION"] = localstack_container["region"]
+    os.environ["AWS_ACCESS_KEY_ID"] = "test"
+    os.environ["AWS_SECRET_ACCESS_KEY"] = "test"
```

Ruff (0.15.0)

[error] Possible hardcoded password assigned to: "AWS_SECRET_ACCESS_KEY"

(S105)